00004-Vulnerability-disclosure-and-embargo-policy/README.md

```diff
@@ -0,0 +1,252 @@
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2020-03-10
+- Status Note: under discussion
+- Supersedes: N/A
+- Start Date: 2020-03-03
+- Tags: security, infrastructure
+
+## Summary
+
+A description of the security vulnerability disclosure process, including embargoes.
+
+## Motivation
+
+Users and researchers who discover security issues in Enarx need to know that
+we welcome their efforts and how we will manage their disclosure. They must
+also be able to have an idea of how we use security embargoes.
+
+In other words:
+- We want:
+  - to encourage reporting of security issues (ie: “Please do attack us!”)
+  - to make it easy to report security issues
+  - to maintain and extend our policy of safe defaults / discouraging people
+  from adopting bad security practices
+  - to recognise and show appreciation for the time spent researching and
+  reporting
+  - to make it possible for people who may be at risk to report issues
+  pseudonymously
+
+- We need:
+  - a clear policy, describing both:
+    - how to report; and
+    - what to expect during the process
+  - a main reporting channel:
+    - secondary channels, should the main one fail/not be available/be
+  overloaded.
+  - a list of people who can be reached directly
+  - a place to list security reports and give attribution
+
+
+## Tutorial
+
+This RFC aims to provide a set of documents (ex. wiki pages) that describe the 
+Enarx vulnerability disclosure policy - including the use of embargoes - and
+the list of security advisories and fixes.
+
+The two pages are "Vulnerability Reporting and Embargo Policy" and "Enarx 
+Security Advisories."
+
+--- 
+### Vulnerability Reporting and Embargo Policy
+The Enarx project aims to be as safe to use as possible and therefore to 
+produce code that is as secure as possible. As such, we welcome any and all 
+research into Enarx and appreciate your efforts to take the time to 
+responsibly disclose any issues you find.
+
+Vulnerabilities may be discovered in:
+1. Enarx code
+2. Enarx dependencies (including compilers)
+3. Enarx documentation
+4. 3rd party hardware or firmware
+
+All security bugs in Enarx should be reported to the security team. 
+To do so, please reach out either by email at security@enarx.dev or on our
+[chat](https://chat.enarx.dev) stating you'd like to report a security
+issue, but **without divulging details**.
+
+We treat both email and chat as a public channels, so simply saying that
+you’re trying to get a hold of someone from the security team is fine.
+
+From then, we will establish a more secure area where the issue can be
+discussed, in the form of a 
+[Github Security Advisory](https://help.github.com/en/github/managing-security-vulnerabilities/about-github-security-advisories).
+
+You will be invited to join this private area to discuss specifics. Doing so 
+allows us to start with a high level of confidentiality and relax it if the 
+issue is less critical, moving to work on the fix in the open.
```

Comment:
Do we have any desire to call out end-to-end encrypted Rocket.Chat as an option here?

Reply:
It's mentioned at the end in the future ideas. The RFC is only in "proposed" state, so let's leave it as is for now, but i like the idea, and we may want to make it more explicit to use OTR when using Rocket Chat to inform the team of a possible security issue.